website/i18n/en.json

```diff
@@ -24,8 +24,7 @@
     },
     "links": {
       "Docs": "Docs",
-      "API": "API",
-      "Help": "Help"
```

Comment:
Why don't we leave the help link and direct to React's Support page?

https://reactjs.org/community/support.html

Reply:
sure I can do that :)